rules/windows/initial_access_dns_server_overflow.toml

```diff
@@ -0,0 +1,47 @@
+[metadata]
+creation_date = "2020/07/16"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/07/16"
+
+[rule]
+author = ["Elastic"]
+description = """
+A specially crafted DNS SIG request can manipulate a known vulnerability in some Windows DNS servers which result in
+Remote Code Execution (RCE) or a Denial of Service (DoS) from crashing the service.
+"""
+false_positives = ["tbd"]
```

Comment:
false_positives = [
    """
    Environments that leverage DNS responses over 60k bytes will identify false positives.
    """,
]